docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   lakefs:
-    image: "treeverse/lakefs:${TAG}"
+    image: "${DOCKER_REG}/treeverse/lakefs:${TAG}"
```

Comment:
did we reach the point when we like docker compose for the end-user and one for the test system?